ubcpi/persistence.py

```diff
@@ -0,0 +1,74 @@
+from submissions import api as sub_api
+
+"""
+answer_item has to be in the format:
+{
+    'answers': [
+        {
+            'vote': 'some option',
+            'rationale': 'some rationale'
+        },
+        {
+            'vote': 'some option',
+            'rationale': 'some rationale'
+        }
+    ]
+}
+"""
+
+ANSWER_LIST_KEY = 'answers'
+
+VOTE_KEY = 'vote'
+RATIONALE_KEY = 'rationale'
+
+
+def get_answers_for_student(student_item):
```

Comment:
Would be great to have better docstrings in this file to understand how these functions and `class Answers` are used.

serialize.py, for example, has excellent documentation.